### DISCUSSION TITLE
Update to Process Definition

### DISCUSSION THREAD

**Discussion by keeysnc** [nodejs/help#4907] [Ideas]
The definition of Process in the node.js documentation is difficult to understand based on the way it's written. 
[](https://nodejs.org/api/process.html#process)

Currently says:
`The process object provides information about, and control over, the current Node.js process. `

Update:
`The process object provides information, and control over, the current Node.js process. `

It's a very simple change but I feel like the current way it's written is slightly confusing.

**Comment by aduh95** [comment]
Do you want to open a PR to propose this change?

**Reply by keeysnc** [reply]
Yes please. =)
